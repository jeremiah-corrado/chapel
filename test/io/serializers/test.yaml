x: 1
y: 2.000000e+00x