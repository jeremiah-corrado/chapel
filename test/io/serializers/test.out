--- !ChildChild
x: 1
y: 42.0
z: 5
...
